core/src/main/clojure/xtdb/sql/plan.clj

```diff
@@ -1747,6 +1808,25 @@
         [:project (mapv :col-sym projected-cols) plan]
         plan))))
 
+(defn- wrap-windows [plan windows]
+  (let [{:keys [windows] :as window-opts} (apply merge-with into (vals windows))
```

Comment:
nit: `(into {} cat (vals windows))`, or similar?

i.e. something that's more obviously getting all the vals into a single map

Reply:
Happy to use something else, but this does something different
```clj
(into {} cat [{:foo [1]} {:foo [2]}])
;; => {:foo [2]}

(apply merge-with into [{:foo [1]} {:foo [2]}])
;; => {:foo [1 2]}
```